imports/ui/components/FreshForm.jsx

```diff
@@ -44,7 +45,32 @@ class FreshForm extends Component {
   };
   setLatLng = latLng => {
     this.setState({ lat: latLng.lat, lng: latLng.lng });
-    console.log(this.state.lat);
+  };
+
+  handleFileSelect = e => {
+    const reader = new FileReader();
+    if (e.target.files.length > 0) {
+      const image = e.target.files[0];
+      reader.readAsDataURL(image);
+      reader.onload = () => {
+        let result = reader.result;
+        this.setState({
+          imageSrc: result
+        });
+      };
+
+      reader.onerror = error => {
+        console.log(error);
+        this.setState({
+          imageSrc: "/images/" + this.state.name + ".png"
+        });
+      };
+    } else {
+      // Image was selected then unselected, need to revert to base image.
+      this.setState({
+        imageSrc: "/images/" + this.state.name + ".png"
+      });
+    }
```

Comment:
Just the setting image part. Let you rewrite to have uploaded image take priority (e.g. uploaded picture of image then set Blueberries name, should show uploaded image not default)

Reply:
Changed prioirty to always use uploaded image.